In_Db2_Machine_Learning/In DB Model Deployment and Management/README.md

```diff
@@ -0,0 +1,337 @@
+# Procedure for using db2 as a backend for model hosting
+
+DB2's Integrated Analytics System allows for an impressive amount of user defined extensions, which can be leveraged to allow db2 to serve as a model management and scoring engine. 
+We can store, manage, update and score models trained in scikit-learn and other python frameworks in DB2. We can also 
+run models trained in Lua, C, R C++. This document will demonstrate a python based example. 
+
+# DB2 Configuration
+
+We will need to store a python class for scoring a generic model in our DB2 server, and make some minor configuration changes. 
+
+Log in to the db2 server. In this case the user is `db2inst1`
+
+`source ~/.bashrc; sudo su - db2inst1`
+
+Make sure there is a python3 installation in your server.
+
+`sudo yum install python3`
+
+Install scikit-learn and joblib
+
+`pip3 install scikit-learn joblib`
+
+Update db2 to use the correct python path
+
+`db2 update dbm cfg using PYTHON_PATH $(which python)`
+
+Restart the database
```

Comment:
```suggestion
Restart the Db2 instance because PYTHON_PATH is not dynamically configurable.
```

Minor point that technically speaking this is a restart of the Db2 instance, not just the database.